src/Cronofy/UserInfo.cs

```diff
@@ -23,12 +23,28 @@ public class UserInfo
         /// </value>
         public string CronofyType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the account's Cronofy account type.
+        /// </summary>
+        /// <value>
+        /// The type of the account.
+        /// </value>
+        public Data CronofyData { get; set; }
+
         /// <summary>
         /// Gets or sets the profiles.
         /// </summary>
         /// <value>The profiles.</value>
         public Profile[] Profiles { get; set; }
```

Comment:
[concern] - Profiles in the UserInfo data model were placed outside of CronofyData block, while on UserInfoResponse they are inside the CronofyData block. From what I can see in our API, profiles belong to the CronofyData block, but chainging it now will mean a breaking-change. My suggestion is to mark the Profiles attribute as deprecated, and copy paste the object into Cronofy's data, to advise our users to start relying on the new one instead.

This can be done on a separate PR, as it's outside of scope for this PR ( CCing @Nevett)